test/index.ts

```diff
@@ -23,6 +23,8 @@ import {javascriptGenerator} from 'blockly/javascript';
 import {load} from './loadTestBlocks';
 import {runCode, registerRunCodeShortcut} from './runCode';
 
+(window as any).Blockly = Blockly;
```

Comment:
Are you intending to check this in? I kinda think we should because I always wind up adding it too, but not sure if it was just left over or not.

Reply:
Oh yeah it was just left over, but we should check it in. it's in the test page so doesn't affect the published plugin, i don't see a downside

Reply:
SGTM!